src/redis/Redis.cpp

```diff
@@ -82,7 +82,7 @@ std::string RedisInstance::loadScript(redisContext* context,
 Redis::Redis(const RedisInstance& instanceIn)
   : instance(instanceIn)
 {
-    // Note, connect with IP, not with hostname
```

Comment:
I removed all instances of this strange habit I have of writing a superfluous "Note, " at the start of comments.